chall07/vsempere.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+	"strconv"
+	"regexp"
+	"log"
+	"io/ioutil"
+	"net/http"
+)
+
+func toHex(dec []byte) string {
+	ri, err := strconv.Atoi(string(dec))
+	if err != nil {
+		log.Fatal(err)
+	}
+	str := strconv.FormatInt(int64(ri), 16)
+	if len(str) < 2 {
+		return "0"+ str
+	} else {
+			return str
+	}
+}
+
+func launchGet(url string) (string, string) {
+	res, err := http.Get(url)
+	if err != nil {
+		log.Fatal(err)
+	}
+	robots, err := ioutil.ReadAll(res.Body)
+	res.Body.Close()
```

Comment:
You could benefit from [defer](https://tour.golang.org/flowcontrol/12) here. It helps you telling the program to run a function when the current function exits. Would that be because of an error, because it returns, because it panics...  
Here you would want to close the body even if `ioutil.ReadAll` panics for some reason:
```suggestion
	res, err := http.Get(url)
	if err != nil {
		log.Fatal(err)
	}
	defer res.Body.Close()
	robots, err := ioutil.ReadAll(res.Body)
```